### ISSUE TITLE
Create the ONE spreadsheet to rule...

### ISSUE THREAD

**Issue by PhiSpel** [joergbrech/Modellbildung-und-Simulation#269]
- [ ] Exercises hierarchy
- [ ] Proofread?
- [ ] formatting ok?
- [ ] licensing ok?
- [ ] ...

**Comment by Florian314** [comment]
https://docs.google.com/spreadsheets/d/1VzA6rIrLjV0GJee_7CQShjTJ-w86xugP6vsIzDmNSGs/edit?usp=sharing

**Comment by PhiSpel** [comment]
@LeaKruehler, can you assign yourself as well to checkmark the exercises you proofread already?

**Comment by LeaKruehler** [comment]
Nice idea with the spreadsheet! 
Assigned myself and added "Korrekturlesen", for me to check, in the spreadsheet.
Im not sure, if the excersices in the spreadsheet are in the correct order, for example the Matlab Einführung is a complete chapter on its own now. I always use the link [tinyurl.com/hbrs-modsim], which is under the about section in the overview in github, to check the current state of the website. Is that even the right way to check it?

**Comment by PhiSpel** [comment]
> Nice idea with the spreadsheet! Assigned myself and added "Korrekturlesen", for me to check, in the spreadsheet. Im not sure, if the excersices in the spreadsheet are in the correct order, for example the Matlab Einführung is a complete chapter on its own now.

That's correct, unfortunately the order changed just after @Florian314 changed it. You can change the order as you wish, the spreadsheet should be independent from it

> I always use the link [tinyurl.com/hbrs-modsim], which is under the about section in the overview in github, to check the current state of the website. Is that even the right way to check it?

That's absolutely right

**Comment by LeaKruehler** [comment]
Just for the record:
I changed it. :)

**Comment by LeaKruehler** [comment]
I included the Spreadsheet to the [wiki](https://github.com/joergbrech/Modellbildung-und-Simulation/wiki), so its easier to find :)
